extensions/styled_components.yaml

```
@@ -4,7 +4,7 @@
 !merge
 contexts: !merge
   expression-begin: !prepend
-    - match: (?=(?:styled|injectGlobal|keyframes){{identifier_break}})
+    - match: (?=(?:styled|createGlobalStyle|injectGlobal|keyframes){{identifier_break}})
```

Will it be ok to add [`css` helper](https://www.styled-components.com/docs/api#css) as well?

```suggestion
    - match: (?=(?:styled|createGlobalStyle|injectGlobal|keyframes){{identifier_break}})
    + match: (?=(?:styled|css|createGlobalStyle|injectGlobal|keyframes){{identifier_break}})
```